lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -220,7 +229,25 @@ std::vector<SetGroupFAR>& SessionState::get_all_far_rules() {
 void SessionState::sess_infocopy(struct SessionInfo* info) {
   // Static SessionInfo vlaue till UPF node value implementation
   // gets stablized.
-  info->state               = magma::lte::Fsm_state_FsmState_CREATING;
+  int curr_state = get_state();
```

Comment:
I would define a separate function like `fsm_state_to_proto`. But why don't we simply use the FSM state proto value directly? Then we don't have to do any conversion code. Maybe add that as a TODO if we want to move forward with that.

Reply:
Noted comments, looks resonable.

Reply:
Sure, I am planning to do this change as part of another PR, that I'll raise.